zign/cli.py

```diff
@@ -40,15 +42,17 @@ def format_expires(token: dict):
 
 @cli.command('list')
 @output_option
-@click.pass_obj
-def list_tokens(obj, output):
+def list_tokens(output):
     '''List tokens'''
     data = get_tokens()
 
     rows = []
     for key, val in sorted(data.items()):
+        access_token = val.get('access_token')
+        if output == 'text' and len(access_token) > 36:
```

Comment:
better use the column width feature of `print_table` instead of truncating manually here